src/pages/journal/journal.effects.ts

```diff
@@ -70,6 +69,7 @@ export class JournalEffects {
     switchMap(
       () => this.callJournalProvider$(JournalRefreshModes.AUTOMATIC).pipe(
         catchError((err) => {
+          // TODO: We don't need to use the store here, just return the action wrapped in an Observable
```

Comment:
Good spot, are these TODOs too hard to change now?

Reply:
Would like to keep this PR focused